guides/common/modules/con_supported-ansible-versions.adoc

```diff
@@ -0,0 +1,5 @@
+[id="supported-ansible-versions_{context}"]
+= Supported Ansible Versions
+
+{Project} uses Ansible as provided by the base operating system of {ProjectServer} and any external {SmartProxies} for remote execution.
```

Comment:
I'm debating the "external" bit here, so I'll write down my thoughts.
* For vanilla Foreman (foreman-deb, foreman-el) there is no concept of "internal" or "external" Smart Proxy. You just have hosts which run a Smart Proxy and they're all the same. This maps poorly to this repo which is still very Katello-centric
* For Katello people do talk about an "internal" Smart Proxy, mostly because it's the single Smart Proxy where Pulp runs in non-mirror mode while all other Content Proxies run Pulp in mirror mode.
* For Satellite this resolves in "external Capsules" but I'd argue that a Capsule is always external

I'm not sure what's best here. One possibility that avoids it is:
```suggestion
{Project} uses Ansible as provided by the base operating system of {ProjectServer} or any {SmartProxies} for remote execution.
```

Reply:
Thanks, I wasn't sure if it was generally appropriate.